draft-ietf-quic-transport.md

```diff
@@ -1232,6 +1232,15 @@ client MUST terminate with a VERSION_NEGOTIATION_ERROR error code if
 version negotiation occurred but it would have selected a different version
 based on the value of the supported_versions list.
 
+When an endpoint accepts multiple QUIC versions, it can potentially interpret
+transport parameters as they are defined by any of the QUIC versions it
+supports.  The version field in the QUIC packet header are only authenticated
+using the transport parameters.  The position and the format of the version
+fields in transport parameters MUST either be identical across different QUIC
+versions, or be unambiguously different in a way that ensures that there is no
+possibility for confusion about their interpretation.  One way that a new format
```

Comment:
suggested rephrase: "be unambiguously different in a way that ensures that there is no possibility for confusion about their interpretation" --> "be unambiguously different to ensure no confusion about their interpretation"